test/unit/dredd-command-test.coffee

```diff
@@ -272,6 +272,61 @@ describe "DreddCommand class", () ->
       it 'should save configuration', ->
         assert.isTrue configUtilsStub.save.called
 
+    describe '"init"', ->
```

Comment:
I don't know what's best practice in that case :( Having several tests just to keep coverage sounds like smell to me, but YMMV. Calling @netmilk for an opinion.

If we are keeping them, can we at least rename [this](https://github.com/apiaryio/dredd/pull/362/files#diff-9583335aa63e76771edff5e02cf4a0c2R257) to

```diff
- describe '"init"', ->
+ describe '"init" (nodejs)', ->
```

and then add the language also to the new ones?


```diff
- describe '"init"', ->
+ describe '"init" (python)', ->
```

etc.